packages/react-reconciler/src/ReactFiberUnwindWork.js

```diff
@@ -331,6 +353,41 @@ function throwException(
         // whole tree.
         renderDidSuspend(root, absoluteTimeoutMs, renderExpirationTime);
 
+        workInProgress.effectTag |= ShouldCapture;
+        workInProgress.expirationTime = renderExpirationTime;
+        return;
+      } else if (
+        enableSuspenseServerRenderer &&
+        workInProgress.tag === DehydratedSuspenseComponent &&
+        shouldCaptureDehydratedSuspense(workInProgress)
+      ) {
+        attachPingListener(root, renderExpirationTime, thenable);
+
+        // Since we already have a current fiber, we can eagerly add a ping listener.
```

Comment:
```suggestion
        // Since we already have a current fiber, we can eagerly add a retry listener.
```

The terminology for "ping" versus "retry" isn't great but that's what we're using everything else